src/third-party/PhishingController.ts

```diff
@@ -95,7 +95,7 @@ export class PhishingController extends BaseController<
 
   private detector: any;
 
-  private handle?: NodeJS.Timer;
+  public lastFetched = 0;
```

Comment:
Seems like this wouldn't be useful outside of this class, so should this be private?